test/libyul/yulOptimizerTests/redundantStoreEliminator/overlapping.yul

```diff
@@ -0,0 +1,19 @@
+{
+    let _1 := 0
+    if callvalue() { revert(_1, _1) }
+    mstore(_1, shl(224, 0x4e487b71))
+    mstore(4, 0x32)
+    revert(_1, 0x24)
+}
+// ====
+// EVMVersion: >=constantinople
+// ----
+// step: redundantStoreEliminator
+//
+// {
+//     let _1 := 0
+//     if callvalue() { revert(_1, _1) }
+//     mstore(_1, shl(224, 0x4e487b71))
+//     mstore(4, 0x32)
```

Comment:
From the example `remove_before_revert` I thought these would be removed

Reply:
The `revert` reads from the memory location `0 - 0x24`, which overlaps both the mstore to `0` as well as the one to `4`, so they cannot be removed.